api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -13,7 +13,9 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
-
+import static org.hamcrest.collection.IsEmptyCollection.empty;
```

Comment:
@dilekamadushan which IDE are you using? Please configure it as described here https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE

static imports are grouped together at the top and imports are sorted alphabetically its all described in the wiki.

the static imports you added have to go above the static imports of `org.junit`

Reply:
@teleivo I am using intelj communtity edition. I will configure it.thanks.